### PR TITLE
Sprint 5/issue 50#user extra information form component

### PR DESCRIPTION
**Pull request by FebernalGober**
# Description

Logic of the components working:
Address receives the address, DateSelector the date of birth and PerfilPhoto brings a photo from the gallery to put it on the profile

the cancel button restarts the form

The Sign Up button submits the form (Logic with Firebase pending)

### CODE REVIEW

File: renta_facil_colima/src/View/Screens/ExtraInfo.jsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+import ExtraInfoState from '../../hooks/ExtraInfoState';
+import ExtraInfostyle from '../../styles/ExtraInfoStyle';
+import DateSelector from '../components/DatePicker';
+import PerfilPhoto from '../components/PerfilPhoto';
+
+const ExtraInfo = () => {
+  const {extraInfoSchemaFormInitialValues, extraInfoSchema} = ExtraInfoState();
+
+  const handleSubmit = (values, {setSubmitting,resetForm}) => {
+    console.log('Formulario enviado con los siguientes valores:', values);
```
Comment (BrandonVG) [inline renta_facil_colima/src/View/Screens/ExtraInfo.jsx]: Remove console.log

File: renta_facil_colima/src/View/Screens/ExtraInfo.jsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                  <Text style={ExtraInfostyle.errorText}>{errors.address}</Text>
+                )}
+              </View>
+
+              <View style={ExtraInfostyle.birthday}>
+                <Field name="birthday">
+                  {({field, form}) => (
+                    <DateSelector
+                      value={field.value} // Valor del campo de fecha
+                      onChange={date => form.setFieldValue('birthday', date)} // Función para actualizar el valor
```
Comment (BrandonVG) [inline renta_facil_colima/src/View/Screens/ExtraInfo.jsx]: Remove comments

File: renta_facil_colima/src/View/components/DatePicker.jsx
Comment (BrandonVG) [inline renta_facil_colima/src/View/components/DatePicker.jsx:1]: FIle name should be the same as component name

File: renta_facil_colima/src/View/components/DatePicker.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+  const toggleDatePicker = () => {
+    setIsDatePickerVisible(!isDatePickerVisible);
+  };
+
+  const handleDateChange = (date) => {
+    setSelectedDate(date);
+    console.log('Fecha seleccionada:', date);
+    toggleDatePicker();
+  };
```
Comment (BrandonVG) [inline renta_facil_colima/src/View/components/DatePicker.jsx]: This data is not going anywhere, you are not sending it to any parent or anything else

File: renta_facil_colima/src/View/Screens/ExtraInfo.jsx (hunk 2 above)
Comment (BrandonVG) [inline renta_facil_colima/src/View/Screens/ExtraInfo.jsx]: In your component you don't receive those props

File: renta_facil_colima/src/View/components/PerfilPhoto.jsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+  const formik = useFormikContext();
+
+
+  const handlePickImage = async () => {
+    const result = await PickImagesFromGallery();
+
+    if (result && !result.didCancel && !result.errorCode) {
+      const uri = result.assets[0].uri;
+      console.log('Ruta de la imagen seleccionada:', uri);
```
Comment (BrandonVG) [inline renta_facil_colima/src/View/components/PerfilPhoto.jsx]: Remove console.log

File: renta_facil_colima/src/hooks/ExtraInfoState.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import * as Yup from 'yup';
+
+const createExtraInfoSchema = () => {
+  return Yup.object().shape({
+    address: Yup.string().required('Address field can not be empty'),
+    profilePicture: Yup.string().notRequired('Profile Picture field can not be empty'),
```
Comment (BrandonVG) [inline renta_facil_colima/src/hooks/ExtraInfoState.js]: if it is notRequired your text is wrong

File: renta_facil_colima/src/styles/DataPickerStyle.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
+import {StyleSheet} from 'react-native';
+import Colors from './Colors';
+
+const styles = StyleSheet.create({
```
Comment (BrandonVG) [inline renta_facil_colima/src/styles/DataPickerStyle.js]: "Styles" as a name is very ambiguous, also, names should be descriptive, also is a very different name in comparison to file name

File: renta_facil_colima/src/styles/ExtraInfoStyle.js
Comment (BrandonVG) [inline renta_facil_colima/src/styles/ExtraInfoStyle.js:1]: Fix indentation and remove unnecessary spaces

**Review by BrandonVG — CHANGES_REQUESTED** [review]
Use eslint and prettier on your files

**Review by AlbertoG22 — APPROVED** [review]

**Review by Lizzflores — APPROVED** [review]

**Review by Tole15 — APPROVED** [review]

**Review by BrandonVG — APPROVED** [review]

**Pull request merged**